certs/swagger.yml

```diff
@@ -109,7 +109,7 @@ definitions:
         description: Key for the client_cert.
       issuing_ca:
         type: string
-        description: 
+        description: TODO
```

Comment:
TODO

Reply:
@mteodor has to fix this, since I don't know what the service does. As said above, "Don't really know. I was just fixing so this thing is error-free. Didn't touch the logic or semantics."

Reply:
I would kindly ask him to help you in this PR. It can't be so long :)

Reply:
``` description: CA Certificate that is used to issue client certs, usually intermediate.```